src/azul/debug.py

```diff
@@ -0,0 +1,67 @@
+from contextlib import (
```

Comment:
rename to `remote_debug`

```suggestion
from contextlib import (
```